ps_facetedsearch.php

```diff
@@ -1484,14 +1484,14 @@ public function getProductByFilters(
                         AND c.active = 1)
                     '.$price_filter_query_out.'
                     '.$query_filters_from.'
-                    WHERE 1 '.$query_filters_where.' GROUP BY cp.id_product');
+                    WHERE 1 '.$query_filters_where.' GROUP BY cp.id_product ORDER BY cp.position ASC');
```

Comment:
ORDER BY is useless, we are return a list of id, we don't care to have it ordered or not